### PR TITLE
Add basic design/styling

### PR DESCRIPTION
**Pull request by simoneduca**
@shaunanoordin or @eatyourgreens mind having a look?

### CODE REVIEW

File: src/presentational/Home.jsx [hunk 1]
```diff
@@ -10,10 +10,19 @@ export default class Home extends Component {
...
     return (
       <Layout>
-        <div className="home-buttons container-fluid">
-          <h1>I am a...</h1>
-          <Link className='btn btn-info' role='button' to='/teachers'>Teacher</Link>
-          <Link className='btn btn-info' role='button' to='/students'>Student</Link>
+        <div className='home'>
+          <div className='jumbotron'>
+           <div className='page-header'>
+            <h1>Welcome to Wildcam Gorongosa Education Portal</h1>
```
Comment (shaunanoordin) [inline src/presentational/Home.jsx]: Two comments here,
- Wording: seems a wee bit odd; recommend either "Welcome to **the** Wildcam Gorongosa Education Portal" _or_ "Welcome to Wildcam **Gorongosa's** Education Portal"
- (Minor) Code formatting: did the tab indent size go from 2 to 1 in this section?

File: src/presentational/Layout.jsx [hunk 2]
```diff
@@ -75,7 +75,8 @@ class Layout extends Component {
         </main>
 
         <footer className='site-footer'>
-          &nbsp;
+          <p className='pull-left'>Zooniverse.org - The universe is too big to explore without you</p>
+          <p className='pull-right'>2016</p>
```
Comment (shaunanoordin) [inline src/presentational/Layout.jsx:79]: I like this footer much better! But we should go whole hog and change that Zooniverse.org label into a full `<a>` link

File: src/presentational/Home.jsx (hunk 1 above)
Comment (simoneduca) [inline src/presentational/Home.jsx]: Yep, I missed the "the", thanks.

File: src/presentational/Layout.jsx (hunk 2 above)
Comment (simoneduca) [inline src/presentational/Layout.jsx:79]: Agree, but the footer will need to change to the full Zooniverse version anyway.

**Comment by shaunanoordin** [comment]
Looks sweet! It's amazing how a few images and the addition of colours make the website look so much better.
- Code looks good; addition of new media files are all OK
- New green colour scheme, background image and replacement of React logo with the Zoo logo all look great.

Tested on Chrome49/Firefox45/Safari9 on MacbookPro+OSX.

Status: **_Fabulous_**
Action: **Merging**

**Pull request merged**